crates/oxc_allocator/src/vec2/mod.rs

```diff
@@ -1295,26 +1295,33 @@ impl<'bump, T: 'bump> Vec<'bump, T> {
 
     /// Retains only the elements specified by the predicate.
     ///
-    /// In other words, remove all elements `e` such that `f(&e)` returns `false`.
-    /// This method operates in place and preserves the order of the retained
-    /// elements.
+    /// In other words, remove all elements `e` for which `f(&e)` returns `false`.
+    /// This method operates in place, visiting each element exactly once in the
+    /// original order, and preserves the order of the retained elements.
     ///
     /// # Examples
     ///
     /// ```
-    /// use bumpalo::{Bump, collections::Vec};
-    ///
-    /// let b = Bump::new();
-    ///
-    /// let mut vec = bumpalo::vec![in &b; 1, 2, 3, 4];
+    /// let mut vec = vec![1, 2, 3, 4];
```

Comment:
`vec!` macro is for `std::vec::Vec`.

```suggestion
    /// use bumpalo::Bump;
    /// let bump = Bump::new();
    /// let mut vec = Vec::from_iter_in([1, 2, 3, 4], &bump);
```

Reply:
Good catch!